docs/reference/mapping/params/position-increment-gap.asciidoc

```diff
@@ -22,7 +22,9 @@ GET /my_index/groups/_search
 {
     "query": {
         "match_phrase": {
-            "names": "Abraham Lincoln" <1>
```

Comment:
These docs say it should still work:
https://www.elastic.co/guide/en/elasticsearch/reference/2.0/query-dsl-match-query.html#query-dsl-match-query-phrase

I haven't tried either one in a while though....